src/can_msg_queue.cpp

```diff
@@ -1,4 +1,4 @@
-
+#include "mbed.h"
```

Comment:
Is mbed needed here? It makes the unit test fail.

If it's needed, it should be escaped with
```
#ifndef UNIT_TEST
```